src/renderer/components/item-object-list/item-list-layout.tsx

```diff
@@ -331,7 +332,7 @@ export class ItemListLayout extends React.Component<ItemListLayoutProps> {
           {this.isReady && info}
         </div>
         {filters}
-        {search}
+        {isSearchable && searchFilters && search}
```

Comment:
seems to make sense to not show the search widget if the list is not searchable.